chalice/chalicelib/audit.py

```diff
@@ -493,6 +493,7 @@ def update_subscriptions():
                 account_name=account["Name"],
                 product_team_id=default_team,
                 active=is_active,
+                in_organisation=True
```

Comment:
So this deals with new accounts being set to True but does not deal with existing accounts being set to True after you add the field and does not deal with former members of the organisation when accounts are shut down rather than suspended. (We may want that to be a manual process rather than an automated one).